content/en/landing/landing.md

```diff
@@ -5,7 +5,7 @@ title: "landing"
 ## Install cairo-lang
 Run this command to install cairo on your machine.
 ```bash
-curl -L https://raw.githubusercontent.com/lambdaclass/cairo-by-example/main/build/installer.sh | bash
+curl -L https://raw.githubusercontent.com/lambdaclass/cairo-by-example/main/build/installer.sh | bash -s latest
```

Comment:
Could we, in some way, default to `latest` so the script can be run directly with no flags for the latest version? What do you think @rcatalan98 ? Is it better with or without `latest` flag?

Reply:
The problem with the latest is that if the language get's a new update it could introduce some breaking changes with the examples we provide. But it looks like Cairo is not going to introduce breaking changes any time soon.

Reply:
It's probably better with the latest or with a specific cairo version sent that is transparent to the user. Because if there is a problem between our examples and the newest version it should be easy to understand that the problem came from there. Does it make sense?